### DISCUSSION TITLE
TFT has a white screen when I upload it via portable Arduino IDE.

### DISCUSSION THREAD

**Discussion by renatoianhez** [Bodmer/TFT_eSPI#2902] [General]
I assembled a pen drive with the portable version of the Arduino IDE. However, when uploading the sketch of my radio, using this pen drive on a new computer, the screen turns on, but is completely blank. The other radio features work normally.
When I upload from an Arduino IDE installed on a computer or through the pen drive on a computer that has the Arduino IDE, the problem does not happen. Why?

**Comment by Bodmer** [comment]
Check the setup file is correctly configured. Run the Read_User_Setup sketch and look at the serial output to check the compiler is picking up the right settings.
